<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSGEditor</class>
 <widget class="QDialog" name="CSGEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор КСГ</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="lblCSGSex">
         <property name="text">
          <string>Пол</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCSGCode">
         <property name="text">
          <string>Код</string>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>350</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="6">
        <widget class="QLineEdit" name="edtEndAgeCount">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMask">
          <string>000; </string>
         </property>
         <property name="maxLength">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblCSGAge">
         <property name="text">
          <string>Возраст</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QComboBox" name="cmbEndAgeUnit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Д</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Н</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>М</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Г</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cmbBegAgeUnit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Д</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Н</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>М</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Г</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="chkCSGInternal">
         <property name="text">
          <string>Внутренний</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3" colspan="5">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>561</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="lblAgeSep">
         <property name="text">
          <string>по</string>
         </property>
         <property name="buddy">
          <cstring>cmbEndAgeUnit</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="chkCSGActiviti">
         <property name="text">
          <string>Активно</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>601</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cmbCSGSex">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>М</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ж</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblCSGNote">
         <property name="text">
          <string>Доп.условие</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="7">
        <widget class="QTextEdit" name="edtCSGNote"/>
       </item>
       <item row="0" column="1" colspan="7">
        <widget class="QLineEdit" name="edtCSGCode"/>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="edtBegAgeCount">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMask">
          <string>000; </string>
         </property>
         <property name="maxLength">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblCSGName">
         <property name="text">
          <string>Название</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="7">
        <widget class="QLineEdit" name="edtCSGName"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblMRBMESGroup">
         <property name="text">
          <string>Профиль</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="7">
        <widget class="CRBComboBox" name="cmbMRBMESGroup"/>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="chkPolyTrauma">
         <property name="text">
          <string>Политравма</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabCSGDiagnosisEx">
       <attribute name="title">
        <string>МКБ</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="CInDocTableView" name="tblCSGDiagnosisEx"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCSGServicesEx">
       <attribute name="title">
        <string>Услуги</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="CInDocTableView" name="tblCSGServicesEx"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>edtCSGCode</tabstop>
  <tabstop>edtCSGName</tabstop>
  <tabstop>cmbCSGSex</tabstop>
  <tabstop>cmbBegAgeUnit</tabstop>
  <tabstop>edtBegAgeCount</tabstop>
  <tabstop>cmbEndAgeUnit</tabstop>
  <tabstop>edtEndAgeCount</tabstop>
  <tabstop>edtCSGNote</tabstop>
  <tabstop>cmbMRBMESGroup</tabstop>
  <tabstop>chkCSGActiviti</tabstop>
  <tabstop>chkCSGInternal</tabstop>
  <tabstop>chkPolyTrauma</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tblCSGDiagnosisEx</tabstop>
  <tabstop>tblCSGServicesEx</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CSGEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CSGEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
